### ISSUE TITLE
[$10][MSFT-63] [Register] "Login" hyperlink foreground and background colors won't meet WCAG 2 AA contrast ratio thresholds

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2778]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member/registration
2. Check the "Login" hyperlink text

**Expected Result**
Expected contrast ratio of 4.5:1

- Passes AA, if you change background color to #2C2C2C. New contrast ratio: 4.53
- Passes AA, if you change text color to #0074C6. New contrast ratio: 4.5

Some people with low vision experience low contrast, meaning that there aren't very many bright or dark areas. Everything tends to appear about the same brightness, which makes it hard to distinguish outlines, borders, edges, and details. Text that is too close in luminance (brightness) to the background can be hard to read.

**Actual Result**

Element has insufficient color contrast of 2.85 (foreground color: #0096ff, background color: #f6f6f6, font size: 9.0pt, font weight: normal).

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AA
**WCAG Success Criterion**: 1.4.3 Contrast (minimum)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: Other

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&sortField=Modified&isAscending=true&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F35

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096291 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096291 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096291 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu PR https://github.com/appirio-tech/accounts-app/pull/221 handles this too

**Comment by Dara-K** [comment]
Let's use #006DEA color for the blue link here.

**Comment by PrakashDurlabhji** [comment]
@veshu @codeMinter Pr updated

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2778 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2778 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2778 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2778 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2778 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2778 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097312 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097312 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by Oanh-and-only-Oanh** [comment]
code review pass PR https://github.com/appirio-tech/accounts-app/pull/221

**Comment by drasticdpk** [comment]
Verified on Chrome browser with suggested color. FYI WAVE tool still showing error.
![image](https://user-images.githubusercontent.com/25297859/62148128-d5db0b00-b318-11e9-8644-3bbccd4c614f.png)

**Comment by nkumar-topcoder** [comment]
Issue is fixed as part of other code merge. Code  `color: #006dea !important;` fix is already available as in PR 221. Please verify @lakshmiathreya @drasticdpk

**Comment by lakshmiathreya** [comment]
LOG IN color fixed as expected. Verified Chrome on MacOS and Win10.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097312<br/><br/>```This is an automated message for crazyk via Topcoder X```
